### DISCUSSION TITLE
[PATCHED]

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#467] [Ideas]
https://thunderstore.io/c/lethal-company/p/ViViKo/BetterDecorPlacement/source/

**Comment by arandomuser0** [comment]
I tested it and you can use it clientside

**Reply by winstxnhdw** [reply]
Others can see when you move the teleporter outside as non-host?

**Reply by arandomuser0** [reply]
yes

**Reply by winstxnhdw** [reply]
Big doubt. Can someone else confirm?

**Reply by joep26020** [reply]
ILL TEST IT SOON sorry caps

**Reply by winstxnhdw** [reply]
I appreciate it!

**Reply by joep26020** [reply]
doesn't work as expected.

**Reply by winstxnhdw** [reply]
Thought so.

**Reply by arandomuser0** [reply]
well it was working before v47, i didn't test it after that

**Reply by winstxnhdw** [reply]
Of course it did. Zeekers only patched building in v49.
